source/application/BioXAS/BioXASAppController.cpp

```diff
@@ -352,15 +354,17 @@ void BioXASAppController::setupScanConfigurations()
 
 void BioXASAppController::setupUserConfiguration()
 {
-	if (userConfiguration_) {
+	if (bioxasUserConfiguration_) {
 
-		connect( userConfiguration_, SIGNAL(loadedFromDb()), this, SLOT(onUserConfigurationLoadedFromDb()) );
+		connect( bioxasUserConfiguration_, SIGNAL(loadedFromDb()), this, SLOT(onUserConfigurationLoadedFromDb()) );
 
-		bool loaded = userConfiguration_->loadFromDb(AMDatabase::database("user"), 1);
+		bool loaded = bioxasUserConfiguration_->loadFromDb(AMDatabase::database("user"), 1);
 		if (!loaded) {
-			userConfiguration_->storeToDb(AMDatabase::database("user"));
+			bioxasUserConfiguration_->storeToDb(AMDatabase::database("user"));
 			onUserConfigurationLoadedFromDb();
 		}
+
+		userConfiguration_ = bioxasUserConfiguration_;
```

Comment:
Is setting this variable going to be handled by the refactor as well?  Seems like an easy place for something to get missed and then having random bugs crop up.